### DISCUSSION TITLE
❓ Questions / Help / Support Можно ли использовать синтезатор для своего канала на ютутбе, например?

### DISCUSSION THREAD

**Discussion by PopkovS** [snakers4/silero-models#231] [Q&A] [answered]
## ❓ Questions and Help

We have a [wiki](https://github.com/snakers4/silero-models/wiki) available for our users. Please make sure you have checked it out first.
Можно ли использовать синтезатор для своего канала на ютутбе, например?

Не нарушу ли я чего?

**Comment by snakers4** [comment, marked as answer]
Можно, в соответствии с лицензией.
